cmd/agent/main.go

```diff
@@ -12,7 +12,6 @@ import (
 func main() {
 	flag.StringVar(&opts.configPath, "config", "/etc/datadog-agent/datadog.yaml", "Path to datadog.yaml config")
 	flag.StringVar(&opts.netConfigPath, "network-config", "/etc/datadog-agent/network-tracer.yaml", "Path to network-tracer.yaml config")
-	flag.StringVar(&opts.ddConfigPath, "ddconfig", "/etc/dd-agent/datadog.conf", "Path to dd-agent config")
```

Comment:
How do we handle providing a flag that does not exist for people that used both `ddconfig` and `config` ?

Reply:
I don't quite understand the question. 

If you provide a flag that's not handled by the binary, it should just be ignored.

Reply:
Just tried I don't think that is the case:

```
$ /opt/datadog-agent/bin/process-agent -ac
flag provided but not defined: -ac
```

Reply:
That's pretty wild...

I'll add it back in but ignore it